### ISSUE TITLE
Simple Tab Groups refuses to open tabs in "Container"

### ISSUE THREAD

**Issue by Zeioth** [cadeyrn/newtaboverride#217]
This addon is also affected by this issue
https://github.com/gdh1995/vimium-c/issues/198

**Comment by cadeyrn** [comment]
Well, it's kind of expected that there can be issues if you use multiple add-ons for changing the way how Firefox handles tabs. Unfortunately I don't have resources to investigate issues that are caused by other add-ons. But if there is a way to do do something within New Tab Override to fix the issue please let me know what should be done and I'll take a look.

**Comment by MathieuDR** [comment]
@Zeioth did you find a satisfactory solution?

**Comment by MathieuDR** [comment]
I think it should be possible as vimium is able to do something similar. I'm not very familiar with this whole extension eco system but @cadeyrn did you learn something over the years?

**Comment by cadeyrn** [comment]
No, I have not invested any time in investigating compatibility with other add-ons. But if anyone has a solution, I accept pull requests.

**Comment by cadeyrn** [comment]
Should be fixed in New Tab Override 18.0. Please test once the new version is available.

**Comment by cadeyrn** [comment]
@Zeioth @MathieuDR Fixed in New Tab Override 18, available now.
